src/app/view/components/dot-navigation/dot-navigation.component.ts

```diff
@@ -42,4 +57,10 @@ export class DotNavigationComponent implements OnInit {
     isActive(id: string) {
         return this.dotNavigationService.isActive(id);
     }
+
+    private isClickedOutsideSidenav(eventTarget: MouseEvent, isSafeToCollapse: boolean): void {
+        if (!this.sideNavEl.nativeElement.contains(eventTarget) && isSafeToCollapse) {
+                this.onMenuLinkClicked.emit(event);
```

Comment:
This logic doesn't make much sense, because:

If I click in the overlay the condition in the line 62 is ```true``` then an event called ```onMenuLinkClicked``` emits and I hide the menu because I'm listening to that, but I didn't click in a menu link like the event ```onMenuLinkClicked``` implied.

The event ```onMenuLinkClicked``` as implied should ONLY emit when I click on a menu link in this component.

But also, we don't need this if we do the comment before.